config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  root "posts#index"
```

Comment:
This looks like it's missing the `PostsController` file, as well as `app/views/posts/index.html.erb`.